src/request-item/request-item.tsx

```diff
@@ -0,0 +1,63 @@
+// NEED TO FIX IMG FILE LOCATION AFTER FOLDER STRUCTURE IS SET
+
+import { h } from 'preact'
+import cn from 'classnames'
+import classes from './request-item.module.css'
+// import girl from '../assets/imgs/girl.jpg'
+import girl from './girl.jpg'
+
+export type RequestItemProps = h.JSX.HTMLAttributes<HTMLAnchorElement> & {
+	requestTitle?: string
+	requestReplies?: number
+	requestDate?: string
+	requesterName?: string
+	requesterRep?: number
+	requestAmt?: number
+	myRequest?: boolean
+	detail?: boolean
+}
+export const RequestItem = ({
+	class: className,
+	...props
+}: RequestItemProps) => (
+	<a class={cn(classes.request, className)} href="">
+		<div class={props.detail ? classes.detail : ''}>
+			<div class={classes.titleRow}>
+				<p class={classes.title}>{props.requestTitle}</p>
+				<div class={classes.replies}>
+					<span>{props.requestReplies}</span>
+				</div>
+			</div>
+			<div class={classes.date}>{props.requestDate}</div>
+			<div class={classes.requesterRow}>
+				<div
+					class={
+						props.myRequest
+							? classes.requesterInfo + ' ' + classes.myRequest
+							: classes.requesterInfo
+					}
```

Comment:
So what you can do here, instead of custom logic for applying multiple classes, is to use `classnames`.

Step 1: import it at the top with `import cn from 'classnames'`. An example can be found here: https://github.com/swarmcity/ui-library/blob/ac8fad446d791407d4719c2c89dfcfecbefd0427/src/button/button.tsx#L2

Step 2: use it to create your classes, so in this case:
```suggestion
					class={cn(classes.requesterInfo, props.myRequest && classes.myRequest)}
```

What this does is:
- Set `classes.requesterInfo` in any case
- If `props.myRequest` in truthy, add the `classes.myRequest` class. If it's not, it'll pass `props.myRequest`'s value to `cn`, which will simply remove it, so you only end up with `classes.requesterInfo`.

Reply:
Excellent! Thank you.